installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -1,33 +1,49 @@
-[id='proc-install-rhdh-ocp-operator_{context}']
-
+[id="proc-install-rhdh-ocp-operator_{context}"]
 = Deploying {product} on {ocp-short} using the Operator
 
-As a developer, you can install {product} on your OpenShift Container Platform using the {product} Operator.
+As a developer, you can deploy a {product} instance on {ocp-short} by using the *Developer* perspective in the web console.
```

Comment:
I would still mention that this procedure is specific to the Operator to differentiate it from the Helm chart procedure and to orient the user so they know they are in the right place. I know it's in the title, but I also think it should be in the intro paragraph.